pkg/util/backoff/policy.go

```diff
@@ -0,0 +1,18 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
So before we had exp backoff, now we have constant backoff? how does that fix the issue? is it just that we're retrying more now, and each backoff in the expbackoff would fail before?